HousingSearchListener.Tests/V1/Gateway/EsGatewayTests.cs

```diff
@@ -149,5 +164,241 @@ public async Task IndexTenureTestCallsEsClient()
+            var tenure = CreateEsPersonTenure();
+
+            var createdResponse = await sut.IndexPerson(person).ConfigureAwait(false);
+            createdResponse.Should().NotBeNull();
+            createdResponse.Result.Should().Be(Result.Created);
+
+            var createdResult = await _testFixture.ElasticSearchClient
+                                           .GetAsync<ESPerson>(person.Id, g => g.Index("persons"))
+                                           .ConfigureAwait(false);
+
+            createdResult.Source.Should().BeEquivalentTo(person);
+
+            var response = await sut.AddTenureToPersonIndexAsync(person, tenure).ConfigureAwait(false);
+
+            response.Should().NotBeNull();
+            response.Result.Should().Be(Result.Updated);
+
+            var result = await _testFixture.ElasticSearchClient
+                                           .GetAsync<ESPerson>(person.Id, g => g.Index("persons"))
+                                           .ConfigureAwait(false);
+
+            var personResult = result.Source;
+
+            personResult.Should().BeEquivalentTo(person, options => options.Excluding(_ => _.Tenures));
+
+            personResult.Tenures.Should().NotBeEmpty();
+            personResult.Tenures.FindLast(_ => _.Id.Equals(tenure.Id)).Should().BeEquivalentTo(tenure);
+
+            Func<Task<UpdateResponse<Person>>> act = async () => await sut.AddTenureToPersonIndexAsync(person, tenure).ConfigureAwait(false);
+
+            act.Should().Throw<ArgumentException>();
+
+            _cleanup.Add(async () => await _testFixture.ElasticSearchClient.DeleteAsync(new DeleteRequest("persons", person.Id))
+                                                                           .ConfigureAwait(false));
+        }
+
+        [Fact]
+        public async Task AddTenureToPersonIndexNonExistPersonResultCreated()
+        {
+            var sut = new EsGateway(_testFixture.ElasticSearchClient);
+            var person = CreatePerson();
+            var tenure = CreateEsPersonTenure();
+
+            var response = await sut.AddTenureToPersonIndexAsync(person, tenure).ConfigureAwait(false);
+
+            response.Result.Should().NotBeNull();
+            response.Result.Should().Be(Result.Created);
+
+            var result = await _testFixture.ElasticSearchClient
+                                           .GetAsync<ESPerson>(person.Id, g => g.Index("persons"))
+                                           .ConfigureAwait(false);
+
+            var personResult = result.Source;
+            personResult.Tenures.Should().BeEquivalentTo(person.Tenures);
+        }
+
+        [Fact]
+        public async Task UpdatePersonAccountTestCallsEsClient()
+        {
+            var sut = new EsGateway(_testFixture.ElasticSearchClient);
+            var person = CreatePerson();
+            var tenure = CreateEsPersonTenure();
+            tenure.Id = person.Tenures.FirstOrDefault().Id;
+
```

Comment:
Same here as with the above test - they look very similar. Perhaps it's worth investigating if we can consolidate them somewhat.